src/Microsoft.OData.Client/Materialization/HttpWebResponseMessage.cs

```diff
@@ -51,7 +50,7 @@ public HttpWebResponseMessage(IDictionary<string, string> headers, int statusCod
         /// Constructor.
         /// </summary>
         /// <param name="httpResponse">HttpWebResponse instance.</param>
-        public HttpWebResponseMessage(HttpWebResponse httpResponse)
+        internal HttpWebResponseMessage(HttpWebResponse httpResponse)

```

Comment:
why the change?

Reply:
This constructor exposes `HttpWebResponse` and so it's not suitable for public use since we no longer support `HttpWebRequest`. It may be possible to safely remove this constructor entirely. It's only used once internally and I think that callsite is only reached when using `HttpWebRequest`.

Reply:
Actually, looking at the PR description again, it appears that I had intended to remove this constructor and had explained the reasoning. Maybe I should go ahead and do that.